### PR TITLE
docs(aio): fix minor typos in deployment guide

### PR DESCRIPTION
**Pull request by cameronbriar**

### CODE REVIEW

**Comment by cameronbriar** [comment]
I signed it!

File: aio/content/guide/deployment.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ can be significantly larger than is strictly necessary to execute the applicatio
 
 The many requests and large payloads mean
 the app takes longer to launch than it would if you optimized it.
-Several seconds may pass (or worse) before the user can see or do anything userful.
+Several seconds may pass (or worse) before the user can see or do anything useful.
```
Comment (gkalpak) [inline aio/content/guide/deployment.md]: But if it is useful for the user, then it is userful 😛

**Review by gkalpak — APPROVED** [review]

**Review by Foxandxss — APPROVED** [review]

**Review by IgorMinar — APPROVED** [review]
thanks!

File: aio/content/guide/deployment.md [hunk 2]
```diff
@@ -628,4 +628,4 @@ Read about how to enable CORS for specific servers at
 
 ## Next steps
  If you want to go beyond the [simple _copy-deploy_](guide/deployment#dev-deploy "Simplest deployment possible") approach,
- read the [AOT Cookbook](cookbook/aot-compiler "AOT Cookbook") next.
\ No newline at end of file
+ read the [AOT Cookbook](cookbook/aot-compiler "AOT Cookbook") next.
```
Comment (petebacondarwin) [inline aio/content/guide/deployment.md]: there is no `cookbook` URL anymore.
This should be `guide/cb-aot-cookbook`

Comment (petebacondarwin) [inline aio/content/guide/deployment.md]: Fancy fixing that in a follow up PR @cameronbriar ?

Comment (cameronbriar) [inline aio/content/guide/deployment.md]: Thanks @petebacondarwin -- please see PR#16165

**Pull request merged**